whois-api/src/main/java/net/ripe/db/whois/api/rest/ReferencedTypeResolver.java

```diff
@@ -81,6 +84,7 @@ public String getReferencedType(final AttributeType attributeType, final CIStrin
                                 // TODO: [AH] for each person or role reference returned, we make an sql lookup - baaad
                                 return rpslObjectDao.findByKey(objectType, value.toString()).getObjectType().getName();
                             } catch (EmptyResultDataAccessException ignored) {
+                                LOGGER.debug("ReferencedTypeResolver {} in {} class", ignored.getMessage(), ignored.getClass().getName());
```

Comment:
"ReferencedTypeResolver" is already printed as the class name at the beginning of the Logger message, no need to add it again